src/resources/views/crud/form_content.blade.php

```diff
@@ -5,7 +5,7 @@
     @include('crud::inc.show_tabbed_fields')
```

Comment:
I don't know if this issue belongs in this file, but it is regarding the FORM. When I get validation errors here's how it looks:

![CleanShot 2023-02-26 at 10 55 45](https://user-images.githubusercontent.com/1032474/221401230-f78bd9e5-04ff-41f1-bd39-e5b3fcd00c38.png)

The validation messages bubble is too high, and doesn't show the validation messages at all.